lib/screens/send_money_screen.dart

```diff
@@ -0,0 +1,262 @@
+import 'package:e_wallet/models/user_model.dart';
+import 'package:e_wallet/providers/user_provider.dart';
+import 'package:e_wallet/screens/contact_list_screen.dart';
+import 'package:e_wallet/util/custom_format_util.dart';
+import 'package:e_wallet/util/decimal_input_formatter.dart';
+import 'package:e_wallet/validators/add_balance_validator.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter/services.dart';
+import 'package:qrscan/qrscan.dart' as scanner;
+
+class Pay extends StatefulWidget {
+  final String routeName = '/send-money';
+  final String sendToAddress;
+
+  Pay([this.sendToAddress = '']);
+
+  @override
+  _PayState createState() => _PayState();
+}
+
+class _PayState extends State<Pay> {
+  var parentContext = new GlobalKey<ScaffoldState>();
+  final _addBalanceFormKey = GlobalKey<FormState>();
+  TextEditingController _amountController = TextEditingController();
+  TextEditingController _descriptionController = TextEditingController();
+  TextEditingController _sendToController = TextEditingController();
+
+  bool addToContacts = false;
+  bool confirmSend = false;
+  bool isLoading = false;
+  int amountIntValue = 0;
+  UserModel user;
+
+  @override
+  Widget build(BuildContext context) {
+    user = Provider.of<UserProvider>(context).userModel;
+    var sendToArg = ModalRoute.of(context).settings.arguments;
+    if (_sendToController.text == null ||
```

Comment:
You may use `.isEmpty()` instead of `== null` for strings.

Reply:
I agree.